tools/build-overrides/build-overrides.sh

```diff
@@ -458,8 +450,7 @@ EOF
                     log_info "Generating ${kieserver_overrides_yaml} ..."
 cat <<EOF > "${kieserver_overrides_yaml}"
 artifacts:
-- name: "KIE_SERVER_DISTRIBUTION_ZIP"
-  target: "kie_server_distribution.zip"
+- name: "rhpam_kie_server_distribution.zip"
```

Comment:
```suggestion
- name: "rhdm_kie_server_distribution.zip"
```

Reply:
thanks for this, too much ctrlC and V issues :D